src/intervals/arithmetic/hyperbolic.jl

```diff
@@ -2,58 +2,93 @@
 # Section 9.1 of the IEEE Standard 1788-2015 and required for set-based flavor
 # in Section 10.5.3
 
-for f in (:sinh, :tanh, :asinh)
+
```

Comment:
There is a TODO (in the first comment of this PR) which mentions that there are still things to be done to have complete parity between `BareInterval` and `Interval`, "e.g, hyperbolic functions". Are hyperbolic functions now complete? (My guess is that they are indeed ready, but otherwise, we should write somewhere what needs to be done.)

Reply:
Oops I will edit this, all the hyperbolic functions are defined.